packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/types.ts

```diff
@@ -31,12 +32,12 @@ export interface FoldersState {
   isLoading: boolean;
   message: string | undefined;
   page: number;
-  onInitialize: () => void;
   pageItems: FolderData[];
   query: string;
-  onSelect: (name: string) => void;
+  onInitialize: () => void;
   onPaginate: (page: number) => void;
   onQuery: (value: string) => void;
   onSearch: () => void;
   onSearchClear: () => void;
+  onSelectFolder: (id: string, folderLocationPath: string) => void;
```

Comment:
Should this be `onFolderSelect`? Pattern that we've generally followed is `onNounVerb`